plugin/CMakeLists.txt

```diff
@@ -7,6 +7,25 @@ if (PLUGIN_DENSE_PARSER)
   target_sources(objxgboost PRIVATE ${xgboost_SOURCE_DIR}/plugin/dense_parser/dense_libsvm.cc)
 endif (PLUGIN_DENSE_PARSER)
 
+if (PLUGIN_RMM)
```

Comment:
Could this possibly go in cmake/FindRMM.cmake?

Reply:
RMM integration is for now an experimental feature, so I've decided to put it as a plugin. We can move it to FindRMM.cmake later.

Reply:
@RAMitchell I believe RMM is adding a CMake export target.  So in later release a simple `find_package` should suffice.